repeat.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+while true
```

Comment:
nit: maybe limit this to 100 or something :sweat_smile:

Reply:
Yeah... A little scary. I feel like there may need to be some volume to trigger this though because it's pretty rare. For a 2 week period I checked logs for, there were only 14 instances. I had previously tried to trigger it with a script with a 1000 for loop, and it never caused it. To be safer though I can increase the max random time so it's not hammering our executors too hard